test/test_word_to_markdown.rb

```diff
@@ -29,8 +29,18 @@ class TestWordToMarkdown < Minitest::Test
   end
 
   should 'parse tables' do
-    skip 'Figure out why there are so many new lines'
```

Comment:
Removed the `skip` for this test, it's working as originally expected except for a missing space after `**Foo**` and `**Bar**`